modules/dxp/apps/segments/segments-experiment-web/src/main/java/com/liferay/segments/experiment/web/internal/display/context/SegmentsExperimentDisplayContext.java

```diff
@@ -308,6 +314,17 @@ private String _getImagesPath() {
 		return _pathToAssets + "/images";
 	}
 
+	private long _getLiveGroupId() throws Exception {
```

Comment:
@cgoncas, I think we can use StagingGroupHelper here, to get the groupId

Reply:
Yeahhhhh!!!, that is totally true

Reply:
@cgoncas, we can also do this:

```

Group group = _stagingGroupHelper.fetchLiveGroup(_themeDisplay.getScopeGroupId());

if (group != null) {
	return group.getGroupId();
}

return _themeDisplay.getScopeGroupId(); 
```
this would cover local and remote staging. What do you think?